components/member-profile/index.js

```diff
@@ -8,12 +10,20 @@ const Profile = (props) => {
   const { data } = useFetch(getDataURL(props.id));
```

Comment:
NIT: destructure id like `const {id}=props`

Reply:
@Shihab-Duzno request you to provide comments on code that i have changed. This line does not seem to be related to my changes.